pkg/kubeapiserver/options/authentication.go

```diff
@@ -228,6 +229,10 @@ func (s *BuiltInAuthenticationOptions) AddFlags(fs *pflag.FlagSet) {
 
 		fs.BoolVar(&s.ServiceAccounts.Lookup, "service-account-lookup", s.ServiceAccounts.Lookup,
 			"If true, validate ServiceAccount tokens exist in etcd as part of authentication.")
+
+		fs.StringVar(&s.ServiceAccounts.IssuerID, "service-account-issuer-id", s.ServiceAccounts.IssuerID, ""+
```

Comment:
probably just call this `service-account-issuer`, and apply StringOrURI validation (if contains `:`, require to be a valid URI, otherwise any value is allowed)